metrics/test/features/dashboard/presentation/widgets/titled_text_test.dart

```diff
@@ -0,0 +1,121 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:metrics/features/dashboard/presentation/widgets/titled_text.dart';
+
+void main() {
+  group("TitledText", () {
+    testWidgets(
+      "can't be created with null title",
+      (WidgetTester tester) async {
+        await tester.pumpWidget(const TitledTextTestbed(
+          title: null,
+        ));
+
+        expect(
+          tester.takeException(),
+          isAssertionError,
+        );
+      },
+    );
+
+    testWidgets(
+      "can't be created with null value",
+      (WidgetTester tester) async {
+        await tester.pumpWidget(const TitledTextTestbed(
```

Comment:
Should we provide `title`, in this case, to make sure that it's failing actually due to value null?